ui-admin/src/portal/siteContent/HtmlPageEditView.tsx

```diff
@@ -9,11 +9,13 @@ import { sectionTemplates } from './sectionTemplates'
 type HtmlPageViewProps = {
   htmlPage: HtmlPage
   readOnly: boolean
+  siteInvalid: boolean
+  setSiteInvalid: (invalid: boolean) => void
   updatePage: (page: HtmlPage) => void
 }
 
 /** Enables editing of a given page, showing the config and a preview for each section */
-const HtmlPageView = ({ htmlPage, updatePage, readOnly }: HtmlPageViewProps) => {
+const HtmlPageView = ({ htmlPage, readOnly, siteInvalid, setSiteInvalid, updatePage }: HtmlPageViewProps) => {
```

Comment:
hmm.. maybe   `hasInvalidContentFormat` or `hasInvalidSection` as a better name for `siteInvalid`?